lib/util.js

```diff
@@ -423,3 +423,21 @@ function compareByGeneratedPositionsInflated(mappingA, mappingB) {
   return strcmp(mappingA.name, mappingB.name);
 }
 exports.compareByGeneratedPositionsInflated = compareByGeneratedPositionsInflated;
+
+/**
+ * Strip any JSON XSSI avoidance prefix from the string (as documented
+ * in the source maps specification), and then parse the string as
+ * JSON.
+ */
+function parseJSON(str) {
+  if (str.startsWith(")]}'")) {
```

Comment:
`startsWith` fails in node 0.10/0.12, sadly: https://travis-ci.org/mozilla/source-map/builds/279505665

Reply:
I wouldn't mind to stop supporting these versions but then I think you'll need a major version change.

Reply:
I'd reuse the previous regexp to do this as a replacement.

Reply:
I'm planning a major version bump anyhow, due to other pending PRs.

0.10 was still seeing new releases last year; so, ugh, I guess we carry on.